41_Redux/parettool/src/Product/Product.jsx

```diff
@@ -0,0 +1,18 @@
+import './Product.css';
+import { useDispatch } from 'react-redux';
+import { addProductToCart } from '../redux/productsSlice';
+
+export default function Product(props) {
+  const dispatch = useDispatch();
+
+  return (
+    props.products.map((product) =>

```

Comment:
Watch how I did with review on Workshop
this iteration should be in `Products` component not single `Product`